### ISSUE TITLE
Jazzy 0.7.0 not compatible with latest XCode8 betas

### ISSUE THREAD

**Issue by bgalamb** [realm/jazzy#645]
Hi
since we upgraded from Xcode8 beta 3 to Xcode8 beta 6 we are seeing that document generation fails. Most probably because there were some changes in the inheritance modifiers e.g. open

it dies always after the last file is parsed and documentation is not generated at all.

```
Parsing XXXConstants.swift (14/14)
/Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/source_declaration/access_control_level.rb:18:in `initialize': cannot initialize AccessControlLevel with 'source.lang.swift.accessibility.fileprivate' (RuntimeError)
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/source_declaration/access_control_level.rb:26:in `new'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/source_declaration/access_control_level.rb:26:in `from_doc'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:182:in `should_document?'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:178:in `block in should_document?'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:176:in `each'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:176:in `any?'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:176:in `should_document?'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:225:in `make_doc_info'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:303:in `block in make_source_declarations'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:269:in `each'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:269:in `make_source_declarations'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:271:in `block in make_source_declarations'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:269:in `each'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:269:in `make_source_declarations'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:538:in `parse'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/doc_builder.rb:141:in `build_docs_for_sourcekitten_output'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/doc_builder.rb:75:in `build'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/bin/jazzy:15:in `<top (required)>'
        from ./doc-resources/tools/bin/jazzy:23:in `load'
        from ./doc-resources/tools/bin/jazzy:23:in `<main>'

```

And a different error as well : 

```
Parsing BaseFormatter.swift (16/16)
/Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/source_declaration/access_control_level.rb:18:in `initialize': cannot initialize AccessControlLevel with 'source.lang.swift.accessibility.open' (RuntimeError)
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/source_declaration/access_control_level.rb:26:in `new'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/source_declaration/access_control_level.rb:26:in `from_doc'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:299:in `block in make_source_declarations'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:269:in `each'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:269:in `make_source_declarations'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:271:in `block in make_source_declarations'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:269:in `each'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:269:in `make_source_declarations'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/sourcekitten.rb:538:in `parse'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/doc_builder.rb:141:in `build_docs_for_sourcekitten_output'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/lib/jazzy/doc_builder.rb:75:in `build'
        from /Users/i040023/.jenkins/workspace/doc-resources/tools/gems/jazzy-0.7.0/bin/jazzy:15:in `<top (required)>'
        from ./doc-resources/tools/bin/jazzy:23:in `load'
        from ./doc-resources/tools/bin/jazzy:23:in `<main>'
```

**Comment by andrew8712** [comment]
I'm having the same issue

**Comment by istx25** [comment]
Fixed as per #647.
